devops/deploy.sh

```diff
@@ -8,31 +8,31 @@ fi
 
 cd /tmp
 git clone https://github.com/lambdaclass/mirra_backend.git --branch ${BRANCH_NAME}
-cd mirra_backend/apps/arena
+cd mirra_backend/
 
 chmod +x devops/entrypoint.sh
 
 mix local.hex --force && mix local.rebar --force
 mix deps.get --only $MIX_ENV
 mix deps.compile
 mix compile
-mix phx.gen.release
-mix release --overwrite
+mix release ${RELEASE} --overwrite
+if [ ${RELEASE} == "game_backend" ]; then
+	mix ecto.migrate
+fi
 
 rm -rf $HOME/mirra_backend
 mv /tmp/mirra_backend $HOME/
 
 mkdir -p $HOME/.config/systemd/user/
```

Comment:
```suggestion
mkdir -p $HOME/.config/systemd/user/

if [ "$(ls ~/.config/systemd/user/*.service 2>/dev/null | wc -l)" -ne 1 ]; then
    exit 1
fi
```

This `if` statement checks if more than one `systemd` service is installed on the systemd service location.